features/admins/admin_works.feature

```diff
@@ -82,6 +82,7 @@ Feature: Admin Actions for Works, Comments, Series, Bookmarks
     When I follow "Hide Bookmark"
       And all indexing jobs have been run
     Then I should see "Item has been hidden."
+      And I should see "Make Bookmark Visible"
```

Comment:
Should I add another check to follow this and confirm visible from regular user end?

Reply:
A regular user should not be able to see a hidden bookmark. But it would be nice to have a check that an admin can see the text of the hidden bookmark.

Reply:
Would you like me to write a test to make sure that a regular user cannot see a hidden bookmark?